src/world/area_tik/tik_24/8A5A80.c

```diff
@@ -1,3 +1,7 @@
 #include "tik_24.h"
 
-INCLUDE_ASM(s32, "world/area_tik/tik_24/8A5A80", func_80240000_8A5A80);
+s32 func_80240000_8A5A80(void) {
+    sprintf((s8*) &wMapShapeName, "tik_18_shape");
```

Comment:
These may be typed wrong, but the casts shouldn't be necessary. Also the rodata string for the second print I think stops after _hit and so tik_25 and tik_23 are other strings after this function. you can define them just like `const char* tik_24_str1 = "tik_25";` etc

Reply:
I have a feeling the & isnt required

Reply:
Looks like `wMapShapeName` is typed wrong (`s32`) in variables.h